js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js

```diff
@@ -1,5 +1,5 @@
 /* jshint unused: false */
-/* global Noty, Blob, window, Joi, sigma, $, tippy, document, _, arangoHelper, frontendConfig, sessionStorage, localStorage, XMLHttpRequest */
```

Comment:
This Blob is actually required at least on Jenkins still.
And I would say it is correct, as Blob is used later in this file.

Reply:
In the browser environment Blob is already predefined, so we don't need to define it yet again. This redefinition is exactly what the newer eslint complains about.